environment.yml

```diff
@@ -12,6 +12,8 @@ dependencies:
 - anaconda::setuptools=27.2.0
 - anaconda::statsmodels=0.8.0
 - conda-forge::altair=1.2.0
+- conda-forge::dask-searchcv
```

Comment:
I think a version number for dask-searchcv would be good. Are you also thinking we would explicitly import dask? So it would be:
```
conda-forge::dask=0.14.3
conda-forge::dask-searchcv=0.0.2
```